script/provider/semantic-tokens.lua

```diff
@@ -42,7 +42,7 @@ local function enable()
                         tokenModifiers = toArray(define.TokenModifiers),
                     },
                     range = true,
-                    full  = false,
+                    full  = true,
```

Comment:
If full is registered, VSCode will resend the full request every time the file is modified. I am not sure if this is a bug, so I am not currently registered for full.

Reply:
I noticed that too; it sends `full` to all language servers, apparently. I'm not sure why; it didn't seem to really slow anything down a whole lot, but it should work with only range.

Reply:
The reason may be that the modification in the visible range will affect the external semantic coloring, and this will be reflected in the minimap.